domain-models-api-interfaces/pom.xml

```diff
@@ -8,7 +8,7 @@
   <artifactId>domain-models-api-interfaces</artifactId>
   <properties>
     <jaxrsVersion>2.0</jaxrsVersion>
-    <generate_routing_context>/rmbtests/test</generate_routing_context>
+    <generate_routing_context>/rmbtests/test,/rmbtests/testForm</generate_routing_context>
```

Comment:
Do we need this line? I removed this whole line and `mvn install` still works for me

Reply:
Good question!

```sh
cd domain-models-api-interfaces
mvn -B install
```

and you'll notice that Rmbtests.java do not have any routingContext calls! So this is needed or something
else that makes it take property from parent pom.

Reply:
> Good question!
> 
> ```shell
> cd domain-models-api-interfaces
> mvn -B install
> ```
> 
> and you'll notice that Rmbtests.java do not have any routingContext calls! So this is needed or something
> else that makes it take property from parent pom.

Do you mean `cd domain-models-runtime`? The `Rmbtests.java` is in `domain-models-runtime` not in `domain-models-api-interfaces`. Anyway, without that line, I can still see below in the console.
```
12:22:31 [] [] [] [] INFO  RestRouting          invoking postRmbtestsTestForm
12:22:31 [] [] [] [] INFO  LogUtil              127.0.0.1:62209 POST /rmbtests/testForm null HTTP_1_1 200 70 11 tid=folio_shared OK
```